.github/workflows/ci-pr.yaml

```diff
@@ -28,6 +28,9 @@ jobs:
     - name: License Check
       run: |
         set -x
+        if [[ -d ".git" ]]; then
```

Comment:
This change is required since the `action/checkout@v2` Github Actions plugin seems to have changed from before. Earlier version seems to be downloading the source as a tar and the new one directly cloning it and setting it up as a git folder before proceeding. 

Thus we have to clear the `.git` to continue validating the next steps.